### PR TITLE
changed the model code to better handle packets

### PR DESCRIPTION
**Pull request by Prathu122**

### CODE REVIEW

**Comment by ramanan-ravi** [comment]
Thanks for the contribution @Prathu122

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/deepfence/FlowMeter
+module github.com/Prathu122/FlowMeter
```
Comment (ramanan-ravi) [inline go.mod:1]: Can you change it back to deepfence?

File: pkg/flowmeter.go [hunk 2]
```diff
@@ -4,8 +4,8 @@ import (
 	"flag"
 	"log"
 
-	"github.com/deepfence/FlowMeter/pkg/constants"
-	"github.com/deepfence/FlowMeter/pkg/packetAnalyzer"
+	"github.com/deepfence/FlowMeter-fix/pkg/constants"
+	"github.com/deepfence/FlowMeter-fix/pkg/packetAnalyzer"
```
Comment (ramanan-ravi) [inline pkg/flowmeter.go:8]: Ca you change it back to FlowMeter?

File: go.mod (hunk 1 above)
Comment (Prathu122) [reply go.mod:1]: Ok I'll do it. I didn't notice there were problems due to the code. I'll change it

File: pkg/flowmeter.go (hunk 2 above)
Comment (Prathu122) [reply pkg/flowmeter.go:8]: Sure

Comment (Prathu122) [reply pkg/flowmeter.go]: I've done the requested changes, please check and let me know

**Pull request merged**